web/packages/teleterm/src/ui/TopBar/AdditionalActions.tsx

```diff
@@ -106,7 +106,7 @@ function useMenuItems(): MenuItem[] {
       Icon: icons.Add,
       onNavigate: () => {
         const doc = documentsService.createAccessRequestDocument({
-          clusterUri: activeRootCluster.uri,
+          clusterUri: workspacesService.getActiveWorkspace().localClusterUri,
```

Comment:
Hmm, this makes me think that we actually shouldn't touch this… It used to accept only root cluster URIs but now it accepts local cluster URIs too?

From what I see, I guess picking the correct cluster in this doc just never worked correctly. The doc used to be always associated with a root cluster, but when adding a new request we'd always look at which cluster is currently selected.

I'd just revert this change, as properly fixing this is probably outside of the scope of this PR.

Reply:
Yeah, I will revert this. 
We may end up not needing this change at all, if we remove resource requests from `NewRequest`. We will only have role requests there (which are always done against the root cluster).